python/arcticdb/storage_fixtures/utils.py

```diff
@@ -92,10 +92,12 @@ def terminate(p: Union[multiprocessing.Process, subprocess.Popen]):
 
 def wait_for_server_to_come_up(url: str, service: str, process: ProcessUnion, *, timeout=20, sleep=0.2, req_timeout=1):
     deadline = time.time() + timeout
-    alive = (lambda: process.poll() is None) if isinstance(process, subprocess.Popen) else process.is_alive
+    if process is not None:
+        alive = (lambda: process.poll() is None) if isinstance(process, subprocess.Popen) else process.is_alive
     while True:
         assert time.time() < deadline, f"Timed out waiting for {service} process to start"
-        assert alive(), service + " process died shortly after start up"
+        if "alive" in locals():
```

Comment:
Do not check `locals()` instead do something like this,

```
alive = lamba: True
```

Reply:
Done